### DISCUSSION TITLE
How to align SVG images properly?

### DISCUSSION THREAD

**Discussion by nickabond** [dompdf/dompdf#2698] [Q&A]
Hi, thanks so much for creating this library, it has been an integral part of my company's workflow!

I'm building a drawing application where the user can draw SVG shapes and things. The alignment of the SVG shapes is very important, but when I put them into DOMPDF the positioning seems to change? I've tried regular units and percentage units and both don't seem to work right. 

I'm referencing the x, y, width, and height attributes of say a `<rect></rect>`

I'm thinking maybe there is some type of conversion I need to do to get them properly aligned maybe because of DPI or something?

**Comment by bsweeney** [comment]
Sorry, I'm having trouble visualizing the issue. Can you provide a sample of the SVG and the PDF? And how it differs from expectations?
